src/streaming/utils/CapabilitiesFilter.js

```diff
@@ -136,6 +136,22 @@ function CapabilitiesFilter() {
             }
             return supported
         });
+
+        // handle scte214:supplementalCodecs
```

Comment:
I would prefer to not have a second iteration over the Representations. I think what we should do:

1. In `filterUnsupportedFeatures` execute `runCodecSupportCheck` with the "default config" but also the config for the `scte214:supplementalCodecs` string. After that, `Capabilities.testedCodecConfigurations` should contain the result for both configurations
2. In `CapabilitiesFilter._filterUnsupportedRepresentationsOfAdaptation` check if at least one of the two valid codec configurations is supported. I would also prefer to not overwrite the `codecs` attribute of `rep`.

Reply:
Not sure how to not overwrite rep.codecs with additional complexity.  Should the array of supplemental codecs be added to the representation model, filtered based on capabilities and SourceBufferSink._getCodecStringForRepresentation() be updated?